QQE-343.md

```diff
@@ -0,0 +1,54 @@
+# QQE-343- Quarkus and non-JSON payload in REST endpoints
+
+JIRA: https://issues.redhat.com/browse/QQE-343
+PR: https://github.com/quarkus-qe/quarkus-test-suite/pull/1599
+
+## Scope of the testing
+The test scope includes validating Quarkus's capability to handle non-JSON payloads in REST endpoints.
+For this testing scope I will choose some of them, for instance XML, YAML and images. 
+Tests will cover both JVM and Native mode executions.
+#### Test scenarios
+
+        1. XML Payload Scenario:
+            * Verify the endpoint's ability to consume and produce XML payloads.
+            * Send a request with an XML payload to the endpoint.
+            * Validate the response, ensuring it matches the expected XML Schema.
+    
+        2. YAML Payload Scenario:
+            * Check the endpoint's ability to return payload from a YAML file.
+            * Send a request with a YAML payload from a file to an endpoint.
+            * Validate the response, ensuring it matches the expected YAML content.
+    
+        3. Image Payload Scenario:
+            * Verify the endpoint's ability to return an image by making a GET request.
+            * Send an image in a multipart form to the endpoint.
+            * Check the endpoint's handling of image upload as part of a multipart form by making a POST request.
+### Impact on testsuites and testing automation:
+New tests will be added inside [http/http-advance-reactive](https://github.com/quarkus-qe/quarkus-test-suite/tree/main/http/http-advanced-reactive) module.
+
+### Impact on resources:
+There will be longer tests execution for jvm and native runs of QE TS.
```

Comment:
```suggestion
There will be roughly XY minutes longer tests execution for JVM and native runs of QE TS.
```
I can't find that comment, but I asked for something more specific than word _longer_.

Reply:
Please look at other TPs, I believe we are trying to give additional time in minutes when possible